tool_collections/taxonomy/kraken2tax/kraken2tax.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="Kraken2Tax" name="Convert Kraken" version="1.0">
+  <description>data to Galaxy taxonomy representation</description>
+    <requirements>
+        <requirement type="package" version="8d245994d7">gb_taxonomy</requirement>
```

Comment:
Ok, not `gnu_coreutils` but we have a package for it :) https://github.com/galaxyproject/tools-iuc/tree/master/packages/package_gnu_awk_4_1_0